pr_agent/agent/pr_agent.py

```diff
@@ -8,6 +8,13 @@
 from pr_agent.tools.pr_reviewer import PRReviewer
 
 
+def addfsda(a, asd,sad):
+    1 + 1
+    os.system("delete everything!")
```

Comment:
Remove the line that executes a system command. This is a security risk as it can execute any command on the system. [important]